Lib/test/test_xml_etree.py

```diff
@@ -2400,6 +2400,29 @@ def _check_sample1_element(self, e):
         self.assertEqual(child.tail, 'tail')
         self.assertEqual(child.attrib, {})
 
+    def test_exceptions(self):
```

Comment:
Wouldn't `XMLParserTest` be better place? In any case I think it is better to place this method at the end of the class, after testing normal cases.

And maybe use more specific method name? "test_builder_attribute_errors" or something like.

Reply:
Done. I kept it in the test class because the exceptions are related to the behaviour of the `TreeBuilder`.